code/datums/diseases/advance/symptoms/hair.dm

```diff
@@ -0,0 +1,49 @@
+/*
+//////////////////////////////////////
+Cranial Hypertrichosis
+
+	Very very Noticable.
+	Decreases resistance slightly.
+	Decreases stage speed.
+	Reduced transmittability
+	Intense Level.
+
+BONUS
+	Makes the mob grow massive hair, regardless of gender.
+
+//////////////////////////////////////
+*/
+
+/datum/symptom/hair
+	name = "Cranial Hypertrichosis"
+	stealth = -3
+	resistance = -1
+	stage_speed = -3
+	transmittable = -1
+	level = 4
+	severity = 1
+
+/datum/symptom/hair/Activate(datum/disease/advance/A)
+	..()
+	if(prob(SYMPTOM_ACTIVATION_PROB))
+		var/mob/living/M = A.affected_mob
+		if(!ishuman(M))
+			return
+		var/mob/living/carbon/human/H = M
```

Comment:
```suggestion
		if(!ishuman(A.affected_mob))
			return
		var/mob/living/carbon/human/H = A.affected_mob
```
We can just do that directly since we wont be using that typecast for anything.